encoding/httpbinding/encode.go

```diff
@@ -29,15 +29,15 @@ type Encoder struct {
 // NewEncoder creates a new encoder from the passed in request. All query and
 // header values will be added on top of the request's existing values. Overwriting
 // duplicate values.
-func NewEncoder(path, query string, headers http.Header) (*Encoder, error) {
+func NewEncoder(path, rawPath, query string, headers http.Header) (*Encoder, error) {
```

Comment:
API break

Reply:
yah this was my change from my raw not-yet-ready code, @syall lets remove this. and try to fix the bug in a diff way